esp/src/src/Utility.ts

```diff
@@ -20,7 +20,7 @@ export function xmlEncode2(str) {
     str = "" + str;
     return str.replace(/&/g, "&amp;")
         .replace(/"/g, "&quot;")
-        .replace(/'/g, "&apos;")
+        .replace(/"/g, "&apos;")
```

Comment:
This change is incorrect. A single-quote character should be encoded as `&apos;`, a double-quote as `&quot;`.